cfpb/consumerfinance.gov
# Add autosize@6.0.1

See background in https://[GHE]/Design-Development/Design-and-Content-Team/issues/263

## Additions

- Add autosize@6.0.1


## How to test this PR

1. Edit a regs page, such as http://localhost:8000/admin/regulations3k/section/edit/9537/, and type in the regs content area. The page should not jump to the top.
